2_adventures/001/chitankensuru/06_other_studying/README.md

```diff
@@ -0,0 +1,4 @@
+##Studying
+
+Will be busy today with learning about process mining and R. So coding will
```

Comment:
Very cool.  What are you reading for R?  I did a [book review](http://rubylearning.com/blog/2012/07/09/book-review-exploring-everyday-things-with-r-and-ruby/) on Ruby and R.